annals-of-botany.csl

```diff
@@ -104,14 +104,24 @@
           </group>
           <text prefix=" " suffix="." macro="publisher"/>
         </if>
+        <else-if type="webpage" match="any">
+          <text macro="title" font-style="italic" prefix=" " suffix=". "/>
+          <text variable="URL" prefix=" " suffix="."/>
+          <date variable="accessed">
+            <date-part name="day" form="numeric" suffix=" "/>
+            <date-part name="month" form="short" suffix=" "/>
+            <date-part name="year" form="long"/>
+          </date>
```

Comment:
@damnation333, is this correct now? I don't see any spaces between the URL and the date (just "."), which probably isn't what you want.